server/config.go

```diff
@@ -18,14 +18,14 @@ type Config struct {
 	JSONRPCAddr *net.TCPAddr
 	GRPCAddr    *net.TCPAddr
 	LibP2PAddr  *net.TCPAddr
-
-	Network    *network.Config
-	DataDir    string
-	Seal       bool
-	Locals     []types.Address
-	NoLocals   bool
-	PriceLimit uint64
-	MaxSlots   uint64
+	Metrics     *Telemetry
```

Comment:
Why don't we call this field `Telemetry` instead of `Metrics`?

Reply:
Not just here, but also in the `helper/config.go`

Reply:
Renamed it Commit [link](https://github.com/0xPolygon/polygon-sdk/pull/216/commits/dec1ff280243cbd04ebf13eb154c4955685b314f)